### PR TITLE
bug(refs T32505): center map

### PR DESCRIPTION
**Pull request by gruenbergerdemos**
**Ticket:** https://yaits.demos-deutschland.de/T32505

Center the map around all drawings and zoom to the combined extent. For that, first we need to get the extent from each layer and then combine them into one extent. After that the map can be fitted to the combined extent.

### How to review/test
Go to Verfahren -> Abschnitte -> Erwiderungen verfassen and click on the "Ortsbezug" button. Make some drawings (line, point, polygon), close it and open it again. The map should be centered around the drawings and zooming in (all drawings should be seen).

### CODE REVIEW

File: client/js/components/procedure/StatementSegmentsList/SegmentLocationMap.vue [hunk 1]
```diff
@@ -233,6 +237,23 @@ export default {
...
     },
 
+    /*
+     * Center the map around all drawings and zoom to the combined extent.
+     */
+    setCenterAndExtent () {
+      const extentPolygon = this.$refs.drawPolygon.getExtent()
+      const extentPoint = this.$refs.drawPoint.getExtent()
+      const extentLine = this.$refs.drawLine.getExtent()
+
```
Comment (salisdemos) [inline client/js/components/procedure/StatementSegmentsList/SegmentLocationMap.vue:247]: can't `this.featuresObject` be used to accomplish thant? As far as I remember, in that var all elements are already combined.

**Review by salisdemos — CHANGES_REQUESTED** [review]

**Review by muellerdemos — COMMENTED** [review]
This solution lgtm from a functional point of view. Since I haven`t worked a lot with maps so far I could not say more

File: client/js/components/procedure/StatementSegmentsList/SegmentLocationMap.vue (hunk 1 above)
Comment (gruenbergerdemos) [reply client/js/components/procedure/StatementSegmentsList/SegmentLocationMap.vue:247]: `this.featuresObject` only contains the coordinates, but not the extent.

Comment (salisdemos) [reply client/js/components/procedure/StatementSegmentsList/SegmentLocationMap.vue:247]: :facepalm: I love fitting wording

**Pull request merged**